style/js/shield_defs.js

```diff
@@ -246,6 +268,8 @@ export function loadShields(shieldImages) {
   };
 
   shields["US:KY"] = roundedRectShield("white", "black", "black", 8, 1, null);
+  shields["US:KY:Business"] = banneredShield(shields["US:KY"], ["BUS"]);
```

Comment:
Looks like we've got another drawing issue on canvas-drawn shields that needs to get resolved.  This time for US:KY:Business=8.

![image](https://user-images.githubusercontent.com/3254090/154783462-4b02775b-316e-41d0-b73e-40e6c0801a4f.png)

Reply:
Okay, branch was just out of sync from main.  Resolved!
![image](https://user-images.githubusercontent.com/3254090/154861677-b9b93fef-e4b4-45ab-adbb-09b72d2fd44c.png)